lib/src/Pos/Web/Server.hs

```diff
@@ -106,23 +106,28 @@ convertHandler nc nodeDBs txpData handler =
     excHandlers = [Catch.Handler catchServant]
     catchServant = throwError
 
-nat
-    :: forall ext ctx m .
-    ( MyWorkMode ctx m, MempoolExt m ~ ext)
-    => m (WebMode ext :~> Handler)
-nat = do
+withNat
+    :: forall ext ctx api m .
+    (MyWorkMode ctx m, MempoolExt m ~ ext, HasServer api '[])
```

Comment:
Oh! Why do we need that `HasServer api '[]` constraint, with the `'[]`?

Reply:
@adinapoli-iohk `'[]`' denotes empty context, and we don't use any context for our handlers indeed -- in this function we use simple `hoistServer`, not `hoistServerWithContext`.

Reply:
Ah, perfect -- Incidentally I have seen this cropping up also in the `servant-quickcheck` package, which made me realise that's safe to use (was originally concerned we were restricting our types too much by doing that 😉  )